src/server/v2/resources/jobs.py

```diff
@@ -555,15 +556,21 @@ def post(self):
 
         current_app.logger.info("%s json_data = %s", log_id, json_data)
 
+        # keep track of optional user input values
+        userDKMS = None
+        if 'require_dkms' in json_data:
+            userDKMS = json_data['require_dkms']
```

Comment:
The variable names here seem inconsistent. require_dkms seems like a boolean, but userDKMS seems like a string specifying the name of the user. Later reading makes me think this is userSpecifiedDKMS.

Reply:
You are correct - I will change var name to make it clearer.